openadapt/record.py

```diff
@@ -1230,6 +1337,20 @@ def record(
         ),
     )
     screen_event_writer.start()
+    
+    browser_event_writer = multiprocessing.Process(
+        target=write_events,
+        args=(
+            "browser",
+            write_browser_event,
```

Comment:
Ideally we would re-use `get_events` here, feel free to implement or add a TODO for now.

Reply:
Can you please add the TODO?

Reply:
Done here: 9ca11a3c16e06a0961107ee69747b55068087566